react-components/src/hooks/useReveal3dResourcesFromScene.ts

```diff
@@ -27,7 +27,7 @@ export const useReveal3dResourcesFromScene = (
   const [resourceOptions, setResourceOptions] = useState<AddResourceOptions[]>([]);
 
   useEffect(() => {
-    if (scene.data === undefined) {
+    if (scene.data === undefined || scene.data === null) {
```

Comment:
I also prefer to use the explicitly for null and undefined instead of using the equality operator ==.
Good!

Reply:
Yeees :vulcan_salute: